index.js

```diff
@@ -42,7 +43,11 @@ WinstonDynamoDB.prototype.log = function (info, callback) {
     debug('log (called by winston)', info);
 
     if (!isEmpty(info.message) || isError(info.message)) {
-        this.add(info);
+        const { message } = info
+
+        for (let i = 0; i < message.length; i += maximalMessageLength) {
+            this.log({ ...info, message: message.slice(i, i + maximalMessageLength) }, () => {});
```

Comment:
shouldn't that be `add`? you are in the context of `log` function so it is recursive 😅